tests/conftest.py

```diff
@@ -57,3 +57,10 @@ def pytest_addoption(parser: Parser):
         action="store",
         help="The OpenStack clouds yaml file for the charm to use.",
     )
+    parser.addoption(
+        "--use-existing-app",
+        action="store",
+        help="The existing app to use. "
```

Comment:
I think there is an extra space after the `.`.

This option is a good idea. Great work here.

Reply:
Thanks, I saw this in another PR: https://github.com/canonical/bind-operator/pull/5#discussion_r1566787699